engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -390,11 +390,12 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
   }
 
   when(WorkflowExecutionAbortingState) {
-    case Event(AbortedResponse(jobKey), stateData) =>
-      workflowLogger.info(s"$tag job aborted: ${jobKey.tag}")
+    case Event(response: BackendJobExecutionResponse, stateData) =>
+      val jobKey = response.jobKey
+      workflowLogger.info(s"$tag job exited with ${response.getClass.getSimpleName}: ${jobKey.tag}")
       val newStateData = stateData.removeBackendJobExecutionActor(jobKey)
       if (newStateData.backendJobExecutionActors.isEmpty) {
-        workflowLogger.info(s"$tag all jobs aborted")
+        workflowLogger.info(s"$tag all jobs exited")
```

Comment:
ToL: I think you were ok before... this is `when(WorkflowExecutionAbortingState)`